### PR TITLE
Configure Dependabot to use Rubygems.org

### PR DESCRIPTION
**Pull request by jturkel**
Fix the Dependabot configuration to use rubygems.org rather than our private gem server.

### CODE REVIEW

**Review by joshbranham — APPROVED** [review]
LGTM assuming the default is to just try rubygems

**Comment by jturkel** [comment]
We'll find out after the PR merges! The doc wasn't super clear but I believe this should work.

**Pull request merged**
